modules/API/API.js

```diff
@@ -282,12 +329,16 @@ class API {
      * been joined.
      *
      * @param {string} roomName - The room name.
+     * @param {string} id - The id of the local user.
+     * @param {Object} props - The display name of the local user.
```

Comment:
I think props has more than display name. Is the display name the most important value?

Reply:
Yes the avatar! I will add it!

Reply:
Done!